src/deploy/RPM_build/packagerpm.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -eou pipefail
+set -x
+
+dir=$(dirname "$0")
+
+SKIP_NODE_INSTALL=1 source $dir/../install_nodejs.sh
+
+noobaaver="$(npm pkg get version | tr -d '"')"
+revision="1"
+releasedate=$(date '+%a %b %d %Y')
+nodever=$(node --version | tr -d 'v')
```

Comment:
don't we need to cauple it with .nvmrc?

Reply:
At the moment this script is run only from within the container which has the node version tied to nvmrc. The reason I went this way was to remove some additional steps from `RPM.Dockerfile` (nothing complex, just that this was simpler). I can change it to use .nvmrc, but do we really expect to be running this script outside of the container?